components/google_sheets/sources/new-updates/new-updates.mjs

```diff
@@ -1,226 +1,25 @@
-import crypto from "crypto";
-import common from "../common.mjs";
+import httpBase from "../common/http-based/sheet.mjs";
+import newUpdates from "../common/new-updates.mjs";
 
 export default {
-  ...common,
+  ...httpBase,
+  ...newUpdates,
   key: "google_sheets-new-updates",
   type: "source",
   name: "New Updates (Instant)",
-  description:
-    "Emit new event each time a row or cell is updated in a spreadsheet.",
-  version: "0.0.25",
+  description: "Emit new event each time a row or cell is updated in a spreadsheet. To use this source with a spreadsheet in a [Shared Drive](https://support.google.com/a/users/answer/9310351), use the **New Updates (Shared Drive, Instant)** source instead.",
+  version: "0.0.26",
   dedupe: "unique",
+  hooks: {
+    ...httpBase.hooks,
+    ...newUpdates.hooks,
+  },
```

Comment:
Since `newUpdates` doesn't have any hooks, and the hooks from `httpBase` will be included anyway, this hooks section can be eliminated.
```suggestion
```